### ISSUE TITLE
Why continue development?

### ISSUE THREAD

**Issue by adminka-root** [giuspen/cherrytree#2419]
Given the number of open issues, the slow pace of development, and the existence of programs like Obsidian that implement modern methods of knowledge base management, the question arises: maybe it's time to stop developing Cherrytree and focus on something else? This tool has functionally outlived its usefulness and will ultimately lead to a negative user experience

**Comment by timcrouse** [comment]
Cherry Tree is far from usable.  It is quite useful in it's current state. 
A couple key use case for it's use:
-The notes are it's own set of self contained notes.  Very easy to utilize on a case by case basis for dedicated project notes.  Easy to transfer to other folks or archive for historical purposes.

-It provides text editing in WYWYG format.  For folks that do not need markdown, Plan text editing is one of the top requests in the Obsidian community. To date thee has been not effort to implement the plain text editing as a core function of Obsidian.  Folks have attempted to create plug-in for this functionality but they seem to come at a a cost of only partially working and or crippling other functionality to mimic plain text editing.

-Cherry Tree is portable out of the box.  Something that tweeking to happen in the Obsidian package.

Best Regards
-Tim C.

**Comment by adminka-root** [comment]
The fact of the matter is that it is extremely problematic to convey to people. Conversion methods to other formats do not work perfectly. For example, converting to HTML does not generate css styles according to the Cherry interface. The native format of the database is “closed”: a black box containing data that can only be effectively displayed by Cherry Tree. Removing personal sub-threads to give to other people to open the file in their Cherry is also a problem, since the base file is dynamically expandable: the data seems to actually remain. You can try exporting individual branches to a separate database file, but this often breaks internal links. Moreover, the process is very slow. Cherry is also not well suited for version control.
As for the WYWYG format, I note that obsidian displays the document this way by default: I can see what I’m writing, and the context menu provides a set of the most commonly used tools for formatting text (for those who are too lazy to learn hotkeys).
So far your argument does not convince me. The only advantage you didn't mention: Cherry Tree is completely free. But the existing shortcomings in the form of poor portability and the lack of simple synchronization, lack of support for mobile devices, and a tree (not graph) structure negate this advantage.

**Comment by timcrouse** [comment]
I am not sure we use CherryTree in the same workflow.  Regarding source control and data structure in contrast to your opinion is one of the key things I find useful.  I use an instance of CherryTree per project.  In that way, all project info is self-contained.  I simply archive the app and the data together and keep it with the project documents.  And if I ever wanted to update the app in that instance its only a  matter of replacing the BIN directory.

FYI I also use Obsidian for general info storage and Excalibdraw (awsome tool) but for the project side, I find CherryTree simple and convenient.

Best Regards
-Tim C.

**Comment by adminka-root** [comment]
Likewise, if you were interested in a "monolith", you could create a filesystem in a regular file, mount it, and store the obsidian files in "solid" storage. True, I don’t understand why the requirement of “monolithicity” arises in principle... In Cherry’s case, you lose the opportunity to subtly compare changes using git diff to understand how your project developed over time. You lose the ability to share notes from your projects with other people because the export features don't work perfectly. Perhaps it is a matter of habit (laziness). The issue of retraining is often difficult for many people. However, if a person uses a pager because for him a smartphone is an incomprehensible magic box without buttons, then this is a problem with the person, not the smartphone.

**Comment by timcrouse** [comment]
Good point about mounting the file and using that as a single project repository.
Something to consider.  If there was an updated portable version of Obsidian that was maintained that would be helpful.
 It would be good to have the excalidraw functionality in a project document set.

**Comment by TriggerDingus** [comment]
I've only been using CherryTree a few years and I committed to it almost immediately after looking over numerous other note apps. CherryTree has advantages for me:

- It's open source. I'll always pick an open source app over a proprietary one, given a reasonable option to do so.
- It allows me to save my data files where I want to put them, without creating numerous database files, parm files, etc and burying them in my home directory. It perplexed me when testing various notes apps that they created so many different supporting files and then wouldn't allow me to select where I wanted them to go. I just want one file (the one I'm editing) and to put it where I want it without a hassle. 
- I don't want my notes app to have network access, period. Many notes apps are designed around network access and I work hard to keep my files offline so network access is a liability in my workflow.
- CherryTree was the first notes app I tested that was completely natural and logical from the moment I opened it. The hiearchical nature works how I work and makes complete sense to me - many other notes apps I tested just didn't have a logical interface for me.

The only thing I desire from CherryTree is perhaps the interface could use a 'freshening' without changing the layout too much. It may look a bit older than newer apps but the features and functionality of CherryTree deep and rich.

My appreciation goes to @giuspen.

**Comment by turkerozturk** [comment]
Cherrytree eşsizdir.
Çok iyi düşünülmüş ve aktif bir yazılım.
Öyle ki, insan sonradan da bir sürü küçük kullanışlı özellik farkediyor.
I respect.
Thanks and regards.
Şu an geliştirme dursa bile, sürekli kullanacağım bir yazılım.
İleride çok daha kullanışlı özellikler ekleneceğinden eminim.
Tekrar belirtmem gerekirse, yeni özellikler eklenmese bile, mükemmel bir yazılım.
Arkadaşlarıma söylüyorum böyle güzel bir yazılım var diye, insanları ikna etmek zor, alışkanlıkları değiştirmek zor. Çok fazla seçenek var gibi görünüyor ilk bakışta. Fakat YOK!. Onlar kaybeder.
Yaşasın Cherrytree.

**Comment by turkerozturk** [comment]
Cherrytree is unique.
A very well thought out and active software.
So much so that people notice a lot of small useful features later on.
I respect.
Thanks and regards.
Even if development stops now, it is a software that I will use constantly.
I'm sure more useful features will be added in the future.
If I have to reiterate, it is an excellent software even if no new features are added.
I tell my friends that there is such a good software, it is difficult to convince people, it is difficult to change habits. At first glance, there seems to be a lot of options. But NO!. They lose.
Long live Cherrytree.

**Comment by instancezero** [comment]
Boy, your deep concern for the experience of "other users" and the effort of the developers is so giving and generous!

Maybe the developers keep working on it because they feel like it. Maybe the open issues are open because they don't feel like addressing them. Maybe it's a hobby, not a business, and they're doing it for their own enjoyment, happy if others like it too. Maybe there's lots of people other than you who are perfectly happy with it and don't care if it doesn't have the latest in "knowledge base management" methods. Maybe if some other tool works better for you, you can go and make valuable contributions like this on their project instead.
